pilot/utils/test_utils.py

```diff
@@ -3,13 +3,13 @@
 
 class TestShouldExecuteStep:
     def test_no_step_arg(self):
-        assert should_execute_step(None, 'project_description') is True

```

Comment:
@LeonOstrez I've had to modify this test. You mentioned that you've fixed an issue here - can you confirm that the updates to these tests are valid? This first one does not seem right